packages/gestalt/src/Masonry.js

```diff
@@ -495,34 +499,35 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
           role="list"
           style={{ height: 0, width: '100%' }}
         >
-          {items
-            .filter((item) => item)
```

Comment:
noticed while I was making this change that we were doing an extra `.filter` for no reason.

Reply:
great callout.  i should have guessed that this was there for a reason.